.github/workflows/run-tests.yml

```diff
@@ -33,6 +33,11 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        # tests run on 20.04 (Qt 5.12), compile test on 20.10 (Qt 5.14)
```

Comment:
I'd be interested to see if we could enable tests on 5.14 too... I think most would pass now.

Reply:
You mean, Qt 5.15?

Reply:
Ubuntu 20.10 has Qt 5.14
With this new workflow, it should be much easier to test with new Qt versions.
@nyalldawson as soon as I have this merged, I'll make a new PR to see what happens with the tests. We could easily have a skip tests file depending on the Qt version later on. That should also ease the move to newer Qt/base image for the future 🤞